manifests/params.pp

```diff
@@ -19,11 +19,6 @@
     $package_name = 'certbot'
     $package_command = 'certbot'
     $config_dir = '/etc/letsencrypt'
-  } elsif $::operatingsystem == 'Ubuntu' and versioncmp($::operatingsystemrelease, '16.04') >= 0 {
```

Comment:
Instead of removing the condition, we may consider package from ubuntu 18.04 and upper enough up to date by default.
```
} elsif $::operatingsystem == 'Ubuntu' and versioncmp($::operatingsystemrelease, '18.04') >= 0 {
```
We probably should have to add Ubuntu 16.04 and 18.04 as supported OSes in `metadata.json`.